runner/src/examples.rs

```diff
@@ -558,6 +558,59 @@ fn run_functions_example(example: &FunctionsExample) -> Step {
     }
 }
 
+pub fn build_functions_example(opt: &RunFunctionsExamples) -> Step {
+    let example: Example = example_toml_files()
```

Comment:
It is not clear to me how this ensures that only functions examples are built here.

Reply:
Thanks. Fixed the condition in `find`, also added a `filter` to makes sure the specified example is a "functions" example.